doc/whats_new/v0.21.rst

```diff
@@ -58,6 +58,21 @@ Support for Python 3.4 and below has been officially dropped.
 :mod:`sklearn.discriminant_analysis`
 ....................................
 
+:mod:`sklearn.semi_supervised`
```

Comment:
This has been inserted in the wrong place. Please make sure to do a manual merge with master.